packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -287,10 +285,8 @@ export async function getEdDSAPublicKey(
     privateKey = fromHexString(privateKey);
   }
 
-  return eddsa
-    .prv2pub(privateKey)
-    .map((p) =>
-      // `F.toObject` converts a point from Montgomery format to a standard one.
-      eddsa.F.toObject(p).toString(16).padStart(64, "0")
-    ) as EdDSAPublicKey;
+  return eddsa.prv2pub(privateKey).map((p) =>
+    // `F.toObject` converts a point from Montgomery format to a standard one.
+    eddsa.F.toObject(p).toString(16).padStart(64, "0")
+  ) as EdDSAPublicKey;
```

Comment:
just checking but are your prettier settings different from the workspace settings?

Reply:
I don't think so. It only happens to one/two files.

FWIW this is what I from the playground. Can you try save or format this document on your end?

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACATnGBXdVVwAmhAzgIYA6+qNAdAA7oBuATPdgEYAUjAlk2XgBpOAE8AlFRp0AtmXpce41AF4AfFOk0A9NtQADAGK0YEAPIcAVnDAx9qSFCZx0MEqjKp6EXrFQAzdAgZVABZaBgAc2CXUQCIdDkYVFMPVBIYMihCMnRCVGg4Wk0tIlIyWmNTC2tbJRMIAGUYdF9IrgBGADZxBjJCZtyYLi6AFgAaVAoQAAZpyWoaZTJ3AFFCABFGgEEABU4AG14wEVEAbhBxkAh6GF5oEmRQXKCAd13chEeUMgPXslEjyuHHQZDAAGssI0yDI4AAZXxwZD+X4kODA0EQqH0MFtZAtbDokBomS8fHoQlXEhtA5wACK2Ag8GRqKJlhIAA9GjT6YzmUgUQc0VcAI58uC7IL0b4gFYAWigcCIREuIBaZF4RygkQAwsE5MhZQcDqrqdradsYC1eBxsPBdi4EYqWUKiQALGAyA4AdTdvHgJBxYDgjS+-v4-tEhrAJCBICYhIAktkEDBGmBWrdttlmqJaS7hSBGBA0d7QfRDYw4GjmEirr4azBJWRIgaBayrjj0DXDTBRPRqxneLdVXxYN7eIQYG7kAAOGZXTBi3iYZutsgFomZDgTqczpAsK7YNEAFTIHG+gsLcBkHDKRDhWUi2BbcEMCSSd21hrIdogIAAXwAoA

Reply:
Nah, it's ok, was just checking.